README.md

```
@@ -1,3 +1,57 @@
-# java-janggi
+# 🚀 장기 미션
 
-장기 미션 저장소
+### 📌 입력
+- 마와 상의 순서를 입력받는다(ex: 마상마상)
+  - 형식에 맞지 않는 경우 예외를 던진다
+- 이동할 기물의 좌표와 좌표가 이동할 기물의 좌표를 입력받는다(ex: 108 88)
+  - 해당 위치로 이동할 수 없는 경우 예외를 던진다
+  - x좌표는 1~10, y좌표는 1~9의 값을 가진다
+
+### 📌 출력
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)